.github/workflows/pr.yaml

```diff
@@ -132,17 +133,25 @@ jobs:
       - name: Install Playwright Browsers
         run: npx playwright install --with-deps ${{ matrix.browser }}
 
+      - name: Set lxd channel env variable
+        id: lxd-env
+        run: |
+          # need to change / to - in lxd channel string for report naming
+          LXD_CHANNEL=$(echo '${{ matrix.lxd_channel }}' | sed 's#/#-#g')
+          echo "LXD_CHANNEL=$LXD_CHANNEL" >> $GITHUB_OUTPUT
+
       - name: Run Playwright tests
-        run: npx playwright test --project ${{ matrix.browser }}
+        run: LXD_CHANNEL=${{ steps.lxd-env.outputs.LXD_CHANNEL }} npx playwright test --project ${{ matrix.browser }}
```

Comment:
I don't really like, that we are using environment variables and the `--project` param to pass info to playwright. Though I see the project param is natively supported, which when used works easily. Where the different backend versions are not and need some custom extension. So not sure if there is a better way.

Reply:
Instead of doing it this way, we can parameterise projects with [this approach](https://playwright.dev/docs/test-parameterize#parameterized-projects). I think this also then gives the projects a good meaning instead of being placeholders.